src/test/java/TopNavigationTest.java

```diff
@@ -0,0 +1,91 @@
+import Components.TopNavigation;
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TopNavigationTest {
+    private static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
+    private static final By TOP_NAVIGATION_BAR_CATEGORY_NAME_FIRST_LEVEL = By.cssSelector(".navigation li.level0 > a> span:not(.ui-menu-icon)");
+    private static final By TOP_NAVIGATION_BAR_CATEGORY_FIRST_LEVEL_LINK = By.cssSelector(".navigation li.level0 > a");
```

Comment:
Usually if something is used in only once place should be not extracted as constant.